frontend/src/App.tsx

```diff
@@ -71,16 +70,15 @@ function App() {
     isAuthenticated: boolean;
   }>(["session"], fetchSessionInfo, { refetchInterval: 60000 });
 
-  const isAuthenticated = isLoading ? false : sessionData?.isAuthenticated;
+  const isAuthenticated =
+    isLoading && !sessionData ? false : sessionData?.isAuthenticated;
 
-  const contextClass = {
```

Comment:
was this deleted because it wasn't being used anywhere @glamboyosa

Reply:
yah. has not been used in a long time.